### ISSUE TITLE
memory exhausted at ClassNameProcessor debug_backtrace after updating to php 8.1

### ISSUE THREAD

**Issue by 0x11DFE** [piwik/piwik#18400]
Just got this issue after updating to PHP 8.1, Error at /plugins/Monolog/Processor/ClassNameProcessor.php:65 any clue?

**Comment by sgiehl** [comment]
@0x11DFE Could you provide a bit more details? When exactly does that error occur? Which Matomo version are you using? Do you have additional plugins installed?

**Comment by 0x11DFE** [comment]
> @0x11DFE Could you provide a bit more details? When exactly does that error occur? Which Matomo version are you using? Do you have additional plugins installed?

I'm using the latest version 4.5, The error occurs on page load it takes few seconds to load (unusual) then returns an error code 5xx. I have few plugins installed but since i cannot access true the webpanel, listing the plugin folder names is the only solution i know.

```
Actions              GeoIp2                Referrers
Annotations          Goals                 Resolution
API                  Heartbeat             RssWidget
BotTracker           ImageGraph            ScheduledReports
BulkTracking         Insights              SecurityInfo
Contents             Installation          SegmentEditor
CoreAdminHome        Intl                  SEO
CoreConsole          IntranetMeasurable    SitesManager
CoreHome             IP2Location           TagManager
CorePluginsAdmin     IP2Proxy              Tour
CoreUpdater          LanguagesManager      Transitions
CoreVisualizations   Live                  TreemapVisualization
CustomDimensions     Login                 TwoFactorAuth
CustomJsTracker      Marketplace           UserCountry
CustomVariables      MobileAppMeasurable   UserCountryMap
DarkTheme            MobileMessaging       UserId
Dashboard            Monolog               UserLanguage
DBStats              Morpheus              UsersManager
DeviceDetectorCache  MultiSites            VisitFrequency
DevicePlugins        Overlay               VisitorInterest
DevicesDetection     PagePerformance       VisitsSummary
Diagnostics          PrivacyManager        VisitTime
Ecommerce            ProfessionalServices  WebsiteMeasurable
Events               Provider              Widgetize
Feedback             Proxy
```

**Comment by Findus23** [comment]
I'd recommend you to update to the 4.6.0 release candidate (https://builds.matomo.org/matomo-4.6.0-rc2.zip) as only Matomo 4.6.0 contains all fixes to make Matomo compatible with PHP 8.1. (the stable release should also come out quite soon)

**Comment by 0x11DFE** [comment]
> I'd recommend you to update to the 4.6.0 release candidate (https://builds.matomo.org/matomo-4.6.0-rc2.zip) as only Matomo 4.6.0 contains all fixes to make Matomo compatible with PHP 8.1. (the stable release should also come out quite soon)

This actually fixed my issue, I am now waiting for plugin updates 😜.
